### ISSUE TITLE
draw.py doesn't work

### ISSUE THREAD

**Issue by almln** [BVLC/caffe#5445]
### Issue summary
When use draw_net_to_file() from pyCaffe get error like "no field _values". After debugging i found that there are "if" statements around line 107 in draw.py like
```python
if len(layer.pooling_param.some_param._values) else 0
```
But this fields are not lists (for pooling layer), and here is error arise.

Small fix make it works fine: just remove "if" and pass params values directly. I can't find where is draw funtions were broken, but it may be connected with protobuf generation and "repeated" values in caffe.proto.

Caffe: master at 793bd96351749cb8df16f1581baf3e7d8036ac37

### Steps to reproduce
```python
net = caffe_pb.NetParameter()
text_format.Merge(open('LeNet.proto').read(), net2)
caffe.draw.draw_net_to_file(net, 'net.png')
```
### Your system configuration
Operating system: Ubuntu 16.04.2
Compiler: gcc 5.4.0
CUDA version: release 7.5, V7.5.17
CUDNN version: 5.0.5
Python 2.7.12
libprotoc 2.6.1

**Comment by almln** [comment]
connected with [https://github.com/BVLC/caffe/issues/3698](url)

**Comment by shelhamer** [comment]
Thanks for reporting the issue! Fixed in #5477 and #5487.
